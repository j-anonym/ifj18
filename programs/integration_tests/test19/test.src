x = inputi()
if x then 
print "true\n"
else
print "false"
end

y= inputf()
a = x + y
print a , "\n"
a = defined? (y)
print a , "\n"
a = defined? (hohoh)
print a , "\n"

