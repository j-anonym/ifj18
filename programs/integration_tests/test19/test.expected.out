true
0x1p+2
